puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -198,15 +193,7 @@ public static void grammar(LexerfulGrammarBuilder b) {
 
         b.rule(PARAMS).is(b.optional(
                 PARAM,
-                b.zeroOrMore(COMMA, PARAM))).skip();
```

Comment:
I dove into it and now better understand.

Moreover ADD_PARAM is not allowed anywhere. For instance it is allowed in RESOURCE_OVERRIDE but not in RESOURCE_INST. So, I'll revert this grammar update and deal with both cases.